config/core/configmaps/autoscaler.yaml

```
@@ -182,3 +182,9 @@ data:
     # allow-zero-initial-scale controls whether either the cluster-wide initial-scale flag,
     # or the "autoscaling.knative.dev/initialScale" annotation, can be set to 0.
     allow-zero-initial-scale: "false"
+
+    # default-max-scale is the cluster-wide default value for the max
```

To be a total pedant, other values in this file that set defaults that can be overridden by annotations (e.g. `initial-scale`, `pod-autoscaler-class`, `target-burst-capacity` etc) aren't prefixed by `default-`, so we should probably stay consistent with that (arguably wrong :-)) choice.

```suggestion
    # max-scale is the cluster-wide default value for the max
```

---

I considered this; I thought maybe the other values were different somehow. Since that's not the case I'll update everywhere.